.github/workflows/update_github_meta.yaml

```diff
@@ -0,0 +1,41 @@
+on:
+  push:
+    branches:
+      - gh-pages
+  schedule:
+    - cron: '15 14 * * *'
```

Comment:
This runs around 14:15 every day. If you want to change the time or frequency, the document is here:

https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#schedule